static/play-index.ts

```diff
@@ -70,10 +88,22 @@ function getVersions(cb: (response: string) => void) {
 	});
 }
 
-function sendRun(source: string, version: string, cb: (response: json) => void) {
-	const payload: string = JSON.stringify({source, version});
+function sendRun(source: string, version: string, opt: string, run: Runner, cb: (response: json) => void, input?: string) {
+	const payload: string = JSON.stringify({source, version, opt, input});
```

Comment:
It seems you're not using this `input` thing, was this intended to be stdin input?